### PR TITLE
Update README.md extend info about env

### PR DESCRIPTION
**Pull request by koliwbr**
in environment-variables section add info about `+` in file name to `$ARGOS_MENU_OPEN` work property

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -255,7 +255,7 @@ Plugin executables are run with the following special environment variables set:
 | Name | Value |
 | --- | --- |
 | `ARGOS_VERSION` | Version number of the Argos extension. The presence of this environment variable can also be used to determine that the plugin is actually running in Argos, rather than BitBar or [kargos](https://github.com/lipido/kargos). |
-| `ARGOS_MENU_OPEN` | `true` if the dropdown menu was open at the time the plugin was run, and `false` otherwise. |
+| `ARGOS_MENU_OPEN` | `true` if the dropdown menu was open at the time the plugin was run, and `false` otherwise. Requires `+` in file name to work. |
```
Comment (p-e-w) [inline README.md:258]: That's incorrect. `ARGOS_MENU_OPEN` is set regardless of whether the filename contains `+` or not.

Comment (koliwbr) [reply README.md:258]: For me it work only if + is in file name

Comment (p-e-w) [reply README.md:258]: I believe you are confusing two separate things. Adding `+` to the filename re-runs the plugin whenever the user opens the menu, *even if the plugin is not scheduled for a re-run based on its update interval.* So the effect of `ARGOS_MENU_OPEN` is visible immediately regardless of the update interval.

But whether or not the `+` is present, `ARGOS_MENU_OPEN` is always set *when the plugin is run.* Of course, if the interval is large and you open the menu at some random point, you may have to wait a long time for the scheduled re-run to happen and the effect of the environment variable to be reflected in the output.

Comment (koliwbr) [reply README.md:258]: I tested it in test.1s.sh and test.1s+.sh
In test.1s.sh it work about one in 5-6 case. Maybe word "required" is incorrect but small note about it maybe by helpful

Comment (p-e-w) [reply README.md:258]: What you're describing sounds like a GNOME Shell bug. If you look in the `_update` function responsible for running the plugin script you will find this:

https://github.com/p-e-w/argos/blob/118056863633ccc1bf927079ed0ba131576232b9/argos%40pew.worldwidemann.com/button.js#L80-L88

As you can see, the `ARGOS_MENU_OPEN` variable is set unconditionally. There is absolutely no connection to the `+` in the filename. And I know for a fact that this used to work just fine, as I have tested it extensively when I added that feature.

There is nothing to add to the documentation here. Instead, I recommend you file an issue describing exactly what is happening in your case, along with the full contents of your script and your Argos and GNOME Shell versions.

**Pull request closed without merging**